docs/source/serving/user-guides.rst

```diff
@@ -7,3 +7,4 @@ Serving User Guides
    update
```

Comment:
minor: the `Getting Started` page for SkyServe is probably too long. We can fix it in the future.